tests/old_tests/test_awe_nlp.py

```diff
@@ -0,0 +1,261 @@
+"""
+--- [ Test: test_awe_nlp.py ] -----------------------------------------------------------
+
+Set of corresponding tests for document features found in awe_nlp.py of writingobserver.
+
+Author: Caleb Scott (cwscott3@ncsu.edu)
```

Comment:
All the other files in this directory are moved from the tests to here. I assume these are all to be deleted. Why does this file show changes instead of a move as well? Will we still be using this file?
The file being in `old_tests` makes me assume it should be deleted, which is causing some confusion.